frontend/packages/dev-console/integration-tests/support/pages/functions/installOperatorOnCluster.ts

```diff
@@ -18,6 +18,14 @@ export const installOperator = (operatorName: operators) => {
       cy.get(operatorsPO.operatorHub.installingOperatorModal).should('be.visible');
       app.waitForLoad();
       cy.byTestID('success-icon').should('be.visible');
+      // Added below cy.request to verify the pipelines subscriptions
+      if (operatorName === operators.PipelinesOperator) {
+        cy.request(
+          'api/kubernetes/apis/operators.coreos.com/v1alpha1/namespaces/openshift-operators/subscriptions/openshift-pipelines-operator-rh',
+        ).then((resp) => {
+          expect(resp.status).toEqual(200);
+        });
+      }
```

Comment:
I still don't think this belongs in the generic function. The specific check should be passed in as param which the pipelines function can do (`verifyAndInstallPipelinesOperator`). Since that function also exists in this file, it's ok for now.

Reply:
Removed this code and replaced with WaitForCRD's [UI checks]